src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  width: $daysPerRow * $dayShift;
+
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: #{$dayGap};
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #eee;
+    padding: $dayPadding;
+    @include square (100px, 100px);
+
+    border: 1px solid #000;
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day}):before {
+        content: "#{$day}";
+      }
+    }
```

Comment:
- nesting :hover 
- bubbling transition from :hove up to class
- change `all` to `transform` better avoid using `all` as it slows down the page load time
```suggestion
    }

    transition: transform 0.5s;

    &:hover {
      background-color: pink;
      cursor: pointer;
      transform: translateY(-20px);
    }
```